### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by jflim98**
# Update documentation

## Summary
Updated the user guide to include random-student command.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -281,7 +289,8 @@ exit
 | Mark all          | ```mark all a/ATTENDANCE_TYPE``` <br> e.g., `mark all a/present`
 | Score             | ```score INDEX cp/SCORE``` <br> e.g., `score 1 cp/5`                                             |
 | Score all         | ```score all cp/SCORE``` <br> e.g., `score all cp/10`
-| Lowest score      | `lowest-score`
+| Lowest score      | `lowest-score`      |
+| Random student    | `random-student`     |
```
Comment (theodoreleebrant) [inline docs/UserGuide.md]: Perhaps having triple backtick to conform to the other entries in the table

**Review by theodoreleebrant — COMMENTED** [review]
One minor change and everything else looks good!

File: docs/UserGuide.md (hunk 1 above)
Comment (jflim98) [reply docs/UserGuide.md]: Fixed!

Comment (theodoreleebrant) [reply docs/UserGuide.md]: Thanks!

**Review by theodoreleebrant — APPROVED** [review]
Looks good to me!

**Pull request merged**